src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
+
+  let!(:bs_requests) do
+    create(:bs_request_with_submit_action,
+           description: 'Please take this',
+           creator: submitter,
+           target_package: target_package,
+           source_package: source_package)
+
+    create(:bs_request,
+           type: 'maintenance_release',
+           source_project: submitter.home_project,
+           creator: submitter)
+
+    create(:set_bugowner_request, creator: submitter)
+
+    create(:bs_request_with_maintenance_release_actions,
+           source_project: source_project,
+           state: :review,
+           creator: submitter)
```

Comment:
Why do you create those non-sensical maintenance release requests if you don't do anything with them in this spec?

Reply:
I populate the data with some level of "pollution", otherwise tailoring the data only with the data I need will not make sure the feature returns the proper set of data once the data is wild and not under control like production is.
Is there a problem having it?